katello/katello-backup

```diff
@@ -82,7 +81,21 @@ optparse = OptionParser.new do |opts|
   @dir = ARGV[0].dup
 end
 
-@databases.delete 'pgsql' if @is_foreman_proxy_content
+def run_cmd(command, exit_codes=[0])
+  result = `#{command}`
+  unless exit_codes.include?($?.exitstatus)
+    STDERR.puts "Failed '#{command}' with exit code #{$?.exitstatus}"
+    cleanup($?.exitstatus)
+  end
+  result
+end
+
+def cleanup(exitstatus=-1)
```

Comment:
@cfouant It would be nice to print out a message saying cleanup actions are being taken so it's clear to the user that these actions are being taken and they know not to ctrl c out